app/controllers/tasks.js

```diff
@@ -83,10 +90,18 @@ export default class TasksController extends Controller {
           this.allTasks[indexOfSelectedTask] = updatedTask;
           this.filterTasksByStatus();
         } else {
-          alert('Failed to update the task');
+          this.toast.warning(
```

Comment:
This will be an error right?? Something failed here, warnings are given when there is a possibility of an error and things may or may not break